packages/charts/react-charting/src/components/DeclarativeChart/PlotlySchemaAdapter.ts

```diff
@@ -273,11 +274,18 @@ export const transformPlotlyJsonToScatterChartProps = (
     const legend = series.name || `Series ${index + 1}`;
     const lineColor = getColor(legend, colorMap);
 
+    let yValues = series.y;
+    if (isAreaChart && series.fill === 'tonexty' && index > 0) {
```

Comment:
dont make any change here. Use tonexty logic in case of tozeroy mode. We will add tozeroy support later as mentioned in the task.